### PR TITLE
TN Committee Scraper

### PR DESCRIPTION
**Pull request by abejburton**

### CODE REVIEW

File: scrapers_next/tn/committees.py
Comment (johnseekins) [inline scrapers_next/tn/committees.py:24]: Bare `except`s are typically considered bad practice. Can we at least do `except Exception` here?

File: scrapers_next/tn/committees.py
Comment (johnseekins) [inline scrapers_next/tn/committees.py:19]: This is quite the `try` block. Any chance we could reduce the number of commands in this block?

**Comment by chrisyamas** [comment]
*Note: New Committee Scraper written by attendee of Open States Summit*

File: scrapers_next/tn/committees.py
Comment (abejburton) [reply scrapers_next/tn/committees.py:19]: I made changes to the try and except block to adjust for your comments. Please let me know if there is anything else you'd like me to change.

**Review by johnseekins — APPROVED** [review]

File: scrapers_next/tn/committees.py
Comment (NewAgeAirbender) [inline scrapers_next/tn/committees.py:34]: Change this to `source` instead of `example_source` so can run without passing the url in the cli

File: scrapers_next/tn/committees.py
Comment (NewAgeAirbender) [inline scrapers_next/tn/committees.py:1]: Remove unused imported classes

**Review by NewAgeAirbender — CHANGES_REQUESTED** [review]
Just some small tweaks but looks great besides! Nice work!

File: scrapers_next/tn/committees.py
Comment (abejburton) [reply scrapers_next/tn/committees.py:1]: For some reason the codespace won't let me resolve those issues. But I fixed the import and example_source lines so this is good to go!

**Comment by johnseekins** [comment]
@abejburton We haven't seen the new commit fixing the last two linting issue. Can you ensure your changes get committed? That will let us merge this scraper.

**Comment by johnseekins** [comment]
Here are your last linting errors:
```
scrapers_next/tn/committees.py:12:1: W293 blank line contains whitespace
scrapers_next/tn/committees.py:18:1: W293 blank line contains whitespace
scrapers_next/tn/committees.py:25:1: W293 blank line contains whitespace
scrapers_next/tn/committees.py:44:1: W391 blank line at end of file
```
(From https://github.com/openstates/openstates-scrapers/actions/runs/3999998876/jobs/6864616201)

**Comment by abejburton** [comment]
Changes are pushed now!

**Review by chrisyamas — APPROVED** [review]
Nice work, @abejburton, on building out the list page and detail page scraper objects, and achieving the level of data quality we need!

While you focused your work on the Senate, your code actually was mostly applicable to both House and Senate (given the similarities of the pages). I made modifications and additions to your original code, and now the scraper is getting all chambers' committees and the subcommittees (which only exist in the house). This wouldn't have been possible to turn around so soon without all your hard work.

It's passing lint checks now too, so once we get a new review and approval from @NewAgeAirbender then this PR will be ready for merge!

**Review by NewAgeAirbender — APPROVED** [review]

**Pull request merged**
